neuralpredictors/regularizers.py

```diff
@@ -17,6 +17,31 @@ def laplace():
     return np.array([[0, -1, 0], [-1, 4, -1], [0, -1, 0]]).astype(np.float32)[None, None, ...]
 
 
+def laplace5x5():
```

Comment:
Where did the definition of these filters come from? It actually appears to LoG, not Laplace filters.